clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -705,10 +705,14 @@ SYCLToolChain::GetCXXStdlibType(const ArgList &Args) const {
 void SYCLToolChain::AddSYCLIncludeArgs(const clang::driver::Driver &Driver,
                                        const ArgList &DriverArgs,
                                        ArgStringList &CC1Args) {
+  // Add ../include/sycl and ../include (in that order)
```

Comment:
Mike, is it important to have this order?
Just want to clarify what's behind this (maybe it would be nice to have a comment inlined).

Reply:
At this point, I don't think it matters, but in general we try to make sure the more specific directory is placed before the general one in case there is any potential overlap.

Reply:
Thanks. Makes sense.